### ISSUE TITLE
Reginald isn't shown upon Alt+Space

### ISSUE THREAD

**Issue by xasx** [GBS3/reginald#2]
Thanks first for this nice program.

I encounter a problem after starting Reginald

Steps: Press Alt+Space
Expected: The Reginald application launcher is shown.
Actual: nothing happens.

I tried to find the place in the code where the shortcut is registered in order to reassign it to a different value so that I can rule out conflicts on my system. However, I couldn't find the code line where it's done (your code is super advanced 💯).

On the other hand, when I assign the Alt+Space to Listary, it works, so there's no general hidden conflict on this computer. Of course, when running Reginald, I have a different shortcut assigned to Listary. And it doesn't work when Listary is shut down. That said, it really should work. Listary is a somewhat similar program.

Unfortunately, there is no tray icon to 1. Indicate the program is running and 2. To invoke the settings window so that I can reassign the shortcut (to set it to a definitively working or more convenient one).

**Comment by giosali** [comment]
First off, thank you so much for opening this issue. I found a fix for it and I'll be releasing an updated version very soon.

> Unfortunately, there is no tray icon to 1. Indicate the program is running and 2. To invoke the settings window so that I can reassign the shortcut (to set it to a definitively working or more convenient one).

That's not good because Reginald does have a tray icon. At first I thought this was due to a JSON parsing error (which would've been a bit of a headache to figure out what the issue is), but instead, the reason why <kbd>Alt</kbd> + <kbd>Space</kbd> wasn't working and why the tray icon wasn't appearing is because the application's %APPDATA% directory was being used for initializing a [FileSystemWatcher](https://learn.microsoft.com/en-us/dotnet/api/system.io.filesystemwatcher?view=net-7.0) object *before* actually being created.

This only affected new users which is just terrible on my part. I could have sworn I tested running the application with the %APPDATA% directory being deleted but I guess I was wrong. I'm sorry for the inconvenience.
